common/directoryBlock/directoryBlockHeader.go

```diff
@@ -16,23 +16,28 @@ import (
 
 var _ = fmt.Print
 
+// DBlockHeader contains information related to a specific directory block.
 type DBlockHeader struct {
-	Version   byte   `json:"version"`
-	NetworkID uint32 `json:"networkid"`
+	Version   byte   `json:"version"`   // The nework version, only supported version seems to be constants.VERSION_0
+	NetworkID uint32 `json:"networkid"` // Three supported networks in constants: MAIN_NETWORK_ID, TEST_NETWORK_ID, LOCAL_NETWORK_ID
 
-	BodyMR       interfaces.IHash `json:"bodymr"`
-	PrevKeyMR    interfaces.IHash `json:"prevkeymr"`
-	PrevFullHash interfaces.IHash `json:"prevfullhash"`
+	BodyMR       interfaces.IHash `json:"bodymr"`       // The Merkle root of the 'body' (entries only) of this directory block
+	PrevKeyMR    interfaces.IHash `json:"prevkeymr"`    // The key Merkle root of the previous directory block
+	PrevFullHash interfaces.IHash `json:"prevfullhash"` // The FullHash of the previous directory block
 
-	Timestamp  uint32 `json:"timestamp"` //in minutes
-	DBHeight   uint32 `json:"dbheight"`
-	BlockCount uint32 `json:"blockcount"`
+	Timestamp  uint32 `json:"timestamp"`  // In theory, the timestamp the directory block was created, in minutes
+	DBHeight   uint32 `json:"dbheight"`   // The directory block height this header information is relevant to
+	BlockCount uint32 `json:"blockcount"` // The number of entry blocks in this directory block
 }
 
+// MaxDirectoryBlockEntryCount is the maximum number of entry blocks in any directory block
+var MaxDirectoryBlockEntryCount = uint32(100000)
```

Comment:
Should be a constant, and should be in the constants.go file

Reply:
moved, and constified